Crestone-Tech/ctechsite
# Ct 33 connect to postgresql server

## Purpose

Connect to SQL server. 

Fixes CT-33

## Changes

Basic skeleton server stuff.

## Steps to Reproduce or Test

- cd ./server
- npm run dev
- observe conformation of connection to DB

```[nodemon] 3.1.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: ts,json
[nodemon] starting `ts-node src/index.ts`
db.config.ts: Connecting to postgres via DB_URL:  postgresql://postgres:postgres@localhost:5432/postgres
Executing (default): SELECT 1+1 AS result
Server is running on port 3000```

![image](https://github.com/user-attachments/assets/4a4cc935-9dd8-439e-b394-737a5de8dc92)
